ballerina/Ballerina.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+authors = ["Ballerina"]
+distribution = "2201.12.0"
+keywords = ["AI", "Devant", "Chunkers"]
+license = ["Apache-2.0"]
+name = "ai.devant"
+org = "ballerinax"
+repository = "https://github.com/ballerina-platform/module-ballerinax-ai.devant"
+version = "1.0.0"
+
+[platform.java21]
+graalvmCompatible = true
```

Comment:
Are we supporting GraalVM in this library. If not we can set it as false

Reply:
graalvm supported by default. since we don't have any native dependencies here